wat

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 log_lines() {
-  local regex='^.*\(starting full system upgrade\|upgraded.*\)$'
+  local installed=''
+  if [ "x$SHOW_INSTALLED" == "x1" ]; then
+      installed='\|installed.*'
```

Comment:
Whoops, didn't even think to identify the indentation being used! I'll go through and amend my commits to match.